experimental/auth-logic/provenance_wrapper.go

```diff
@@ -0,0 +1,39 @@
+// Copyright 2022 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package authlogic contains logic and tests for interfacing with the
```

Comment:
Perhaps remove this comment, and add a comment below `package authlogic` explaining what `provenanceWrapper` does.

Reply:
Actually I think this is correct, i.e. the package comment should be before the package definition (https://go.dev/doc/effective_go#commentary).

Then of course it would be nice to also have a comment for `provenanceWrapper` :)

Reply:
`authlogic` is a multi-file package. In this case it does not make sense to repeat the package documentation in every file. A more reasonable approach: https://yaqs.corp.google.com/eng/q/5774342963855360#a5741031244955648

Reply:
I think this is not the case described in that answer since there is no `auth-logic.go` ... hmm... I am happy to do whatever you folks want; I don't have much of an opinion about this. For now I'll just leave the comment that exists and add an additional comment about this specific wrapper just below the package line.